contracts/mesh-provider/src/contract.rs

```diff
@@ -37,6 +38,7 @@ pub fn instantiate(
         slasher: None,
         lockup: deps.api.addr_validate(&msg.lockup)?,
         unbonding_period: msg.unbonding_period,
+        list_validators_retries_remaining: LIST_VALIDATORS_RETRIES,
```

Comment:
Part of me would store this in it's own Item, as the rest of those change very rarely and only on admin message.
Kind of "separation of concerns" (but this is a nit, open to counterarguments)